package.json

```diff
@@ -24,14 +24,14 @@
   ],
   "license": "BSD",
   "dependencies": {
-    "underscore": "~1.4.4",
     "async": "~0.2.5",
-    "cli": "~0.4.4-2",
-    "redis": "~0.8.2", 
-    "js-yaml": "~2.1.0"
+    "cli": "^1.0.1",
+    "js-yaml": "^4.1.0",
+    "redis": "~0.8.2",
+    "underscore": "^1.13.2"
   },
   "devDependencies": {
-    "jshint": "~1.0.0",
-    "jasmine-node": "~1.3.0"
+    "jasmine-node": "^1.16.0",
```

Comment:
Why doesn't this package use `3.0.0` but specifically this version?
```
$ npm outdated
Package       Current  Wanted  Latest  Location                   Depended by
async          0.2.10  0.2.10   3.2.3  node_modules/async         nutcracker-agent
jasmine-node   1.16.0  1.16.2   3.0.0  node_modules/jasmine-node  nutcracker-agent
redis           0.8.6   0.8.6   4.0.3  node_modules/redis         nutcracker-agent
npm notice 
npm notice New minor version of npm available! 8.1.2 -> 8.5.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.5.0
npm notice Run npm install -g npm@8.5.0 to update!
npm notice
```

Reply:
First of all, minimizing breaking changes. There's simply no good reason to abandon v1 at this point. Moreover, >1.16.0 requires vulnerable dependencies.